src/Uno.UI.FluentTheme.v1/Resources/Version1/Priority01/Common_themeresources_v1.xaml

```diff
@@ -35,21 +35,10 @@
             <Thickness x:Key="TextControlThemePadding">10,6,6,5</Thickness>
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
-   
+
     <!-- DateTimeFlyoutBorderPadding is defined since RS5. set it here to ensure it's always defined. -->
     <Thickness x:Key="DateTimeFlyoutBorderPadding">0</Thickness>
 
-    <!-- Default DataTemplate used for the IInputValidationControl.ErrorTemplate property -->
```

Comment:
This interface was removed at some point and there is no `ErrorTemplate` property anywhere now.